File: PROJETOS/20180712-useacabeca-c/cap2/barco.c
#include<stdio.h>

void go_south_east(int *lat, int *lon) {
    *lat = *lat - 1;
    *lon = *lon + 1;
}

int main() {
    int lat = 32;
    int lon = -64;
    go_south_east(&lat, &lon);
    printf("Estamos agora em (%i, %i)!\n", lat, lon);
    return 0;
}

File: PROJETOS/20180712-useacabeca-c/cap2/barco.go
package main

import "fmt"

func go_south_east(lat, lon *int) {
	*lat = *lat - 1
	*lon = *lon + 1
}

func main() {
	var lat int = 32
	var lon int = -64
	go_south_east(&lat, &lon)
	fmt.Printf("Estamos agora em (%d, %d)!\n", lat, lon)
}
